app/views/admin/creator_settings/new.html.erb

```diff
@@ -37,7 +37,7 @@
       <br>
       <%= render "form" %>
       <div class="crayons-field mt-6 align-left">
-        <%= submit_tag "Finish", class: "crayons-btn btn--primary" %>
+        <%= submit_tag "Finish", class: "crayons-btn btn--primary w-100 m:w-100" %>
```

Comment:
Refactor (blocking): No need for any width CSS utility classes or the `flex` CSS utility class. The parent, `.crayons-field` has its display set to flex with a flex-flow of column.

![image](https://user-images.githubusercontent.com/833231/145596039-1460a740-8887-458f-bc55-52cf02fabe5a.png)

```suggestion
        <%= submit_tag "Finish", class: "crayons-btn btn--primary" %>
```